.changeset/eleven-laws-shout.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Okaaaaaayyyy... Let's not do a changeset - I think this changelog entry is going to have to be fully manual, I don't think this will work as it is.